Google Scholar.js

```diff
@@ -535,7 +535,7 @@ function scrapeAll(doc, itemObjs) {
 	
 	if (item.citeUrl) {
 		ZU.doGet(item.citeUrl, function(text) {
-			var m = text.match(/href="(\/scholar.bib\?[^"]+)/);
+			var m = text.match(/href="((https?:\/\/[a-z\.]*)?\/scholar.bib\?[^"]+)/);
```

Comment:
I see here for example https://scholar.googleusercontent.com/scholar.bib?q=info:fhnZqP6GwcwJ:scholar.google.com/&output=citation&scisig=AAGBfm0AAAAAV5vi2gv3GQZF_rjeQQJxk68ks7pvIsSl&scisf=4&ct=citation&cd=-1&hl=de , i.e. the links are not relative.